test/test_train_mp_imagenet.py

```diff
@@ -234,6 +234,9 @@ def train_imagenet():
       summary_writer=writer)
   loss_fn = nn.CrossEntropyLoss()
 
+  if FLAGS.profile:
```

Comment:
Thanks! Is the issue that we should start the server in each process? What's the error that you got?

Reply:
I was getting:
```
2023-02-18 01:13:15.291150: E tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:601] PERMISSION_DENIED: open(/dev/accel0): Operation not permitted: Operation not permitted; Couldn't open device: /dev/accel0; [/dev/accel0] 
2023-02-18 01:13:15.291243: F tensorflow/compiler/xla/xla_client/xrt_local_service.cc:58] Non-OK-status: tensorflow::NewServer(server_def, &server_) status: PERMISSION_DENIED: open(/dev/accel0): Operation not permitted: Operation not permitted; Couldn't open device: /dev/accel0; [/dev/accel0] 
https://symbolize.stripped_domain/r/?trace=7f6dad9178eb,7f6dadc4172f,7f6c57f9c78f,7f6c4b193f5c,7f6c4b16322b,7f6c4b161837,7f6c4b12baee,7f6c4b136297,7f6dadc3e996,e8486666000c298b&map=adfa3e6ba6ce64ea6fe734e394f841fc976ace9b:7f6c482bf000-7f6c5b9636f0 
*** SIGABRT received by PID 2568225 (TID 2568225) on cpu 53 from PID 2568225; stack trace: ***
PC: @     0x7f6dad9178eb  (unknown)  raise
    @     0x7f6c4768739a       1152  (unknown)
    @     0x7f6dadc41730  1019697760  (unknown)
    @     0x7f6c57f9c790         16  tsl::internal::LogMessageFatal::~LogMessageFatal()
    @     0x7f6c4b193f5d        768  xla::XrtLocalService::XrtLocalService()
    @     0x7f6c4b16322c        832  xla::XrtComputationClient::MaybeCreateLocalService()
    @     0x7f6c4b161838        880  xla::XrtComputationClient::XrtComputationClient()
    @     0x7f6c4b12baef       1344  xla::ComputationClient::Create()
    @     0x7f6c4b136298         32  std::call_once<>()::{lambda()#2}::__invoke()
    @     0x7f6dadc3e997  (unknown)  __pthread_once_slow
    @ 0xe8486666000c298c  (unknown)  (unknown)
https://symbolize.stripped_domain/r/?trace=7f6dad9178eb,7f6c47687399,7f6dadc4172f,7f6c57f9c78f,7f6c4b193f5c,7f6c4b16322b,7f6c4b161837,7f6c4b12baee,7f6c4b136297,7f6dadc3e996,e8486666000c298b&map=adfa3e6ba6ce64ea6fe734e394f841fc976ace9b:7f6c482bf000-7f6c5b9636f0,67de04c284e7576daa26061396ba8e0e:7f6c3c034000-7f6c4789eea0 
E0218 01:13:15.357272 2568225 coredump_hook.cc:414] RAW: Remote crash data gathering hook invoked.
E0218 01:13:15.357286 2568225 client.cc:278] RAW: Coroner client retries enabled (b/136286901), will retry for up to 30 sec.
E0218 01:13:15.357291 2568225 coredump_hook.cc:512] RAW: Sending fingerprint to remote end.
E0218 01:13:15.357296 2568225 coredump_socket.cc:120] RAW: Stat failed errno=2 on socket /var/google/services/logmanagerd/remote_coredump.socket
E0218 01:13:15.357299 2568225 coredump_hook.cc:518] RAW: Cannot send fingerprint to Coroner: [NOT_FOUND] Missing crash reporting socket. Is the listener running?
E0218 01:13:15.357302 2568225 coredump_hook.cc:580] RAW: Dumping core locally.
2023-02-18 01:13:16.455105: W tensorflow/tsl/profiler/lib/profiler_session.cc:110] Profiling is late by 1953740 nanoseconds and will start immediately.
E0218 01:13:21.147955 2568225 process_state.cc:784] RAW: Raising signal 6 with default behavior
Traceback (most recent call last):
  File "test/test_train_mp_imagenet.py", line 314, in <module>
    xmp.spawn(_mp_fn, args=(FLAGS,), nprocs=FLAGS.num_cores)
  File "/workspaces/work/pytorch/xla/torch_xla/distributed/xla_multiprocessing.py", line 400, in spawn
    result = torch.multiprocessing.start_processes(
  File "/home/ptxla/.local/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 198, in start_processes
    while not context.join():
  File "/home/ptxla/.local/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 140, in join
    raise ProcessExitedException(
torch.multiprocessing.spawn.ProcessExitedException: process 0 terminated with signal SIGABRT
```